release-notes.md

```diff
@@ -1,3 +1,54 @@
+# Quasar-17.17.0
+## Changes EmailEventNotification co-editors or co-publishers changed templates ##
+* Issue Type: Task
+* Issue ID: SUP-28334
+
+#### Configuration ####
+None.
+
+### Deployment scripts ###
+If the event notification template does not exist in the system use "add scripts", otherwise use "update scripts".
```

Comment:
change the section to something like:
### Deployment scripts ###
If the event notification template does not exist in the system use "add scripts", otherwise use "update scripts".
First replace all tokens in the XML file below and remove ".template" from the file name, then run the php deployment script

Add scripts:
Co editor
    - deployment/updates/scripts/xml/2021_12_16_addCoEditorsChangedEmailNewCoEditorsEmailNotification.template.xml
    - deployment/updates/scripts/2021_12_16_deploy_add_email_event_notification_Co_Editor_Changed.php
Co publisher
    - deployment/updates/scripts/xml/2021_12_16_addCoPublishersChangedEmailNewCoPublishersEmailNotification.template.xml
    - deployment/updates/scripts/2021_12_16_deploy_add_email_event_notification_Co_Publisher_Changed.php

Update scripts:
Co editor -
    - deployment/updates/scripts/xml/2021_12_16_updateCoEditorsChangedEmailNewCoEditorsEmailNotification.template.xml
    - deployment/updates/scripts/2021_12_16_deploy_update_email_event_notification_Co_Editor_Changed.php
Co publisher -
    - deployment/updates/scripts/xml/2021_12_16_updateCoPublishersChangedEmailNewCoPublishersEmailNotification.template.xml
    - deployment/updates/scripts/2021_12_16_deploy_update_email_event_notification_Co_Publisher_Changed.php